### ISSUE TITLE
CMD Carry Out String Command Parameter - false negatives due to trailing space before wildcard in search [BUG]

### ISSUE THREAD

**Issue by cxosmo** [splunk/security_content#2928]
The [detections/endpoint/cmd_carry_out_string_command_parameter.yml](https://github.com/splunk/security_content/blob/bc1824d5c155fe3d25733a76f924f770e06c8a55/detections/endpoint/cmd_carry_out_string_command_parameter.yml#L18) detection includes a trailing space in its search logic (`Processes.process="* /c *"`):

```
| tstats `security_content_summariesonly` min(_time) as firstTime max(_time) as lastTime from datamodel=Endpoint.Processes where `process_cmd` AND Processes.process="* /c *" by Processes.dest Processes.user Processes.parent_process_name Processes.parent_process Processes.process_name Processes.original_file_name Processes.process Processes.process_id Processes.parent_process_id 
| `drop_dm_object_name(Processes)` 
| `security_content_ctime(firstTime)` 
| `security_content_ctime(lastTime)` 
| `cmd_carry_out_string_command_parameter_filter`
```

Since `cmd.exe` can be successfully executed without a space between the `/c` parameter and the subsequent command(s), this  produces false negatives for users that execute `cmd.exe` in this way (e.g. to circumvent this detection). 

For example, executing `cmd.exe` as "expected", with a trailing space: 

```
C:\Windows\System32>cmd.exe /c ver

Microsoft Windows [Version 10.0.22621.2861]
```

Executing `cmd.exe` without a trailing space is still successful:

```
C:\Windows\System32>cmd.exe /cver

Microsoft Windows [Version 10.0.22621.2861]
```

I'd recommend to change the search to remove the trailing space before the wildcard as updated below. This should catch both use cases successfully (e.g. `cmd.exe /c whoami` and `cmd.exe /cwhoami`):

```
| tstats `security_content_summariesonly` min(_time) as firstTime max(_time) as lastTime from datamodel=Endpoint.Processes where `process_cmd` AND Processes.process="* /c*" by Processes.dest Processes.user Processes.parent_process_name Processes.parent_process Processes.process_name Processes.original_file_name Processes.process Processes.process_id Processes.parent_process_id 
| `drop_dm_object_name(Processes)` 
| `security_content_ctime(firstTime)` 
| `security_content_ctime(lastTime)` 
| `cmd_carry_out_string_command_parameter_filter`
```

**Comment by patel-bhavin** [comment]
@cxosmo : thank you for the details in here for showing both successful executions, we will fix this detection in the next release v4.18.0 :)

**Comment by josehelps** [comment]
This issue was released as part of 4.19.0 than you again for sharing!
